Bottom: 871f88d4156aa9a384cd2196975402a904858632
Top:    31965f1e2b237ded8bbecbe586d3314fcd15da18
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-11 16:29:54 -0500

Move ansible role vars to role defaults

This allows other parts of the playbook i.e. host vars, command line the
ability to override these defaults when necessary.


---
diff --git a/roles/docker-openssl/vars/main.yml b/roles/docker-openssl/defaults/main.yml
similarity index 100%
rename from roles/docker-openssl/vars/main.yml
rename to roles/docker-openssl/defaults/main.yml
